src/main/utils.js

```diff
@@ -1,6 +1,10 @@
-import { Menu } from 'electron'
+import { app, Menu } from 'electron'
 
 export const getMenuItem = menuName => {
   const menus = Menu.getApplicationMenu()
   return menus.items.find(menu => menu.label === menuName)
 }
+
+export const getHomePath = () => {
```

Comment:
Hi, thank you for your PR, I think you can provide a getPath uitl just call the app.getPath.

```javascript
export const getPath = directory => {
    return app.getPath(directory)
}
```